lib/cli/index.js

```diff
@@ -384,13 +380,6 @@ var _ = require('lodash'),
             prog: program
         });
 
-        parser.addArgument(['--version'], {
-            help: 'Prints the version and exits',
```

Comment:
As per discussion with @czardoz we should keep the duplicates so that help text show up for legacy help text.